app/models/subcase.js

```diff
@@ -20,10 +20,12 @@ export default class Subcase extends ModelWithModifier {
   @belongsTo('concept') agendaItemType;
 
   // inverse: null or serialize: false is used for possible concurrency issues when saving without reloading possible stale relations.
-  @hasMany('agenda-activity', { inverse: null }) agendaActivities;
+  @hasMany('agenda-activity', { inverse: null, serialize: false })
```

Comment:
Can we try to only keep `serialize: false` and remove `inverse: null`?  Same for `decision-activity`.

Since both are 1-to-many relations, I even assume that `inverse: null` and `serialize: false` can both be removed as ember-data will always save the relation at the `belongsTo` side. Can we give that a try?

Reply:
`serialize: false` is mainly for concurrent `subcase` saves while someone else is postponing.
We don't hit the "modified by" in that case since postponing sets subcase on `agenda-activity` inversely.
maybe `inverse: null` can go, in some relations it has a use but thinking of a scenario where it is needed is difficult.
In some cases we don't want the ember store to update the inverse for reasons that are escaping my mind right now.
Some of those `inverse: null` are also legacy code, so I'm also in favor of trying to remove that one.

Reply:
put this in a separate PR for now.
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1519